src/contact.js

```diff
@@ -0,0 +1,5 @@
+const form = document.getElementById("form");
+
+form.addEventListener('dblclick', function onClick(event) {
+  document.body.style.backgroundColor = 'pink';
```

Comment:
Apreciez că încerci să mă păcălești.

Dar aș prefera să faceți lucruri mai utile în JS.

Reply:
Merci pentru feedback, ai dreptate, am sa incerc sa gasesc niste evenimente mai utile pana marti.
P.S. nu am incercat sa te pacalesc, nu am avut prea mare inspiratie la evenimente zilele astea si nici prea mult timp la dispozitie :)
M-am intins cam mult cu timpul alocat sa fac de la 0 paginile, dar am preferat sa le fac (in loc sa folosesc paginile de la proiectul trecut) ca sa am mai multe lucruri de pus la portofoliu (cand vor fi gata).